app/sesame/automod/emotes.go

```
@@ -46,23 +49,127 @@ func (e *EmoteSet) Has(code string) bool {
+		b.observer = o
+	}
+	if p, ok := x.(interface {
+		PurgeTokens(channel uint64, tokens []string)
+	}); ok {
+		b.purger = p
+	}
+	g.extra.Store(b)
+}
+
+// extras loads the current provider box, nil when none was installed.
+func (g *Gate) extras() *extraBox { return g.extra.Load() }
```

golangci-lint (2.12.2)

[error] func (*Gate).extras is unused

(unused)